krcal/map_builder/config_HEcal.conf

```diff
@@ -2,7 +2,7 @@ folder             = '{folderin}'
 file_in            = '{filein}'
 file_bootstrap_map = '$ICDIR/database/test_data/kr_emap_xy_100_100_r_6573_time.h5'
 file_out_map       = '{fileoutmap}'
-file_out_hists     = '{fileouthist}'
+monitoring_path    = '{folderouthists}'
```

Comment:
stupid comment, not meant to change it, but, why not `{folderoutmonitoring}` or something similar, since there are things that are not hist?